lib/daru/dataframe.rb

```diff
@@ -560,6 +560,11 @@ def replace_values old_values, new_value
       self
     end
 
+    def reset_index!
+      @index = Daru::Index.new(Array.new(size) { |i| i })
+      self
+    end
```

Comment:
A you sure it is what required by #206? Doesn't quite like pandas' [reset_index](https://github.com/SciRuby/daru/issues/206)